backup/README.md

```diff
@@ -102,6 +102,15 @@ tracking the same db (as we do: production and AWS) they will differ by
 instance_name. And if they are tracking different databases they will also
 differ by database.
 
+### Tests
+To test the metrics Flask app, run
+```
+sudo -H install ./metrics/bin/requirements.txt ./metrics/bin/requirements-dev.txt
+pytest
+```
+
+On Teamcity we run these tests inside a docker container using `./metrics/scripts/teamcity.sh`
```

Comment:
This script needs be run from the root of the repo (and therefore needs to be run as `./backup/metrics/scripts/teamcity.sh`)